docs/FeatureManagement/FeatureFlag.v2.0.0.schema.sample.scenarios.md

```diff
@@ -0,0 +1,41 @@
+| 18     | Theta             | user="Adam"                       | True            | Targeting Filter, 74% default rollout Adam is part of it.            |
+| 19     | Theta             | user="Adam" groups=["Stage1"]     | True            | Targeting Filter, group is not part of default rollout calculation, no change            |
+| 20     | Theta             | user="Adam" groups=["Stage2"]     | True            | Targeting Filter, group is not part of default rollout calculation, no change            |
+| 21     | Theta             | user="Adam" groups=["Stage3"]     | True            | Targeting Filter, group is not part of default rollout calculation, no change            |
+| 22     | Theta             | user="Brittney" groups=["Stage1"] | False           | Targeting Filter, 74% default rollout Brittney is not part of it            |
+| 23     | Theta             | user="Brittney" groups=["Stage2"] | False           | Targeting Filter, group is not part of default rollout calculation, no change            |
+| 24     | Theta             | user="Brittney" groups=["Stage3"] | False           | Targeting Filter, group is not part of default rollout calculation, no change            |
+| 25     | Theta             | user="Brittney" groups=["Stage4"] | False           | Targeting Filter, group is not part of default rollout calculation, no change            |
+| 26     | Iota              | N/A                               | True            | Feature Flag with two feature filters, first returns true, so it's enabled.            |
+| 27     | Kappa             | N/A                               | True            | Feature Flag with two feature filters, second returns true, so it's enabled.            |
+| 28     | Lambda            | N/A                               | True            | Feature Flag with two feature filters and requirement type specified as Any. Second filter returns true            |
+| 29     | Mu                | N/A                               | False           | Feature Flag with two feature filters and requirement type specified as Any. Neither filter returns true            |
+| 30     | Nu                | user="Adam"                       | False           | Feature Flag with two feature filters and requirement type specified as All. Only the first fitler returns true          |
+| 31     | Xi                | user="Adam"                       | True            | Feature Flag with two feature filters and requirement type specified as All. Both filters return true            |
+
+## Extra Test Scenario
+
+Update the json to have the default rollout percentage of Theta to be 75% instead of 74% and run the test scenarios again. The expected result for tests 22-25 should now be True, this validates we are using the same fomula for our rollout calculation.
```

Comment:
```suggestion
Update the json to have the default rollout percentage of Theta to be 75% instead of 74% and run the test scenarios again. The expected result for tests 22-25 should now be True, this validates we are using the same formula for our rollout calculation.
```